spec/models/user_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe "validations" do
+    USER_FIELDS = %i(email country).freeze
+    subject { FactoryGirl.build(:user) }
+
+    USER_FIELDS.each do |attr|
+      it { is_expected.to validate_presence_of attr }
+    end
+
+    it { is_expected.to validate_length_of :email }
+  end
+
+  context "active user" do
+    it "#active_for_authentication?" do
```

Comment:
Nice assertions here! I'd suggest re-jigging them a little to match the usual pattern of:
```
describe "#some_method" do
  context "some context" do
    it "does something we expect" do
       expect....
    end
  end
end
```

Let me know if you'd like me to clarify further on that :)